src/UKHO.ERPFacade.CleanUp.WebJob/Services/CleanUpService.cs

```diff
@@ -25,25 +26,30 @@ public CleanUpService(ILogger<CleanUpService> logger,
         }
 
         public void CleanUpAzureTableAndBlobs()
-        {            
-            CleanUpBlobsAndTablesRelatedToEESEvent();
+        {
+            CleanUpEvents(Constants.S57EventTableName, Constants.S57EventContainerName);
         }
 
-        //Private methods       
-        private void CleanUpBlobsAndTablesRelatedToEESEvent()
+        private void CleanUpEvents(string tableName, string eventContainerName)
         {
-            _logger.LogInformation(EventIds.FetchEESEntities.ToEventId(), "Fetching all EES entities from azure table");
-            var entities = _azureTableReaderWriter.GetAllEntityForEESTable();
+            _logger.LogInformation(EventIds.FetchEESEntities.ToEventId(), "Fetching all records from azure table {0}", tableName);
```

Comment:
Can we use variable names in string interpolation to keep it consistent with other logs.